jobs/env/ci-kubernetes-e2e-gci-gce-sd-logging-k8s-resources.env

```diff
@@ -0,0 +1,7 @@
+### job-env
+KUBE_LOGGING_DESTINATION=gcp
+
+TEST_CLUSTER_LOG_LEVEL=--v=2
+
+# Ingest logs against new "k8s_container" and "k8s_node" resources.
+LOGGING_STACKDRIVER_RESOURCE_TYPES=new
```

Comment:
You probably want to enable metadata agent as well here?

Reply:
With `$LOGGING_STACKDRIVER_RESOURCE_TYPES` set to `new`, `enable_metadata_agent` will be true (provided by the new config map). As for the env `$STACKDRIVER_METADATA_AGENT_URL`, it's not set explicitly because we will use `"http://${HOSTNAME}:8799"` as the default value when `$STACKDRIVER_METADATA_AGENT_URL` is not set.

Reply:
I mean, by default the metadata agent will not run in the cluster. You need to set ENABLE_METADATA_AGENT=stackdriver.

Reply:
Oh! I see. For some reaason I thought that's enabled by default. Just added `ENABLE_METADATA_AGENT=stackdriver`.

Reply:
I also saw `KUBE_ENABLE_METADATA_AGENT`. Should that be the env name instead?

```
$ cat ./ci-kubernetes-e2e-gce-stackdriver.env
### job-env

KUBE_ENABLE_CLUSTER_MONITORING=stackdriver
KUBE_ENABLE_METADATA_AGENT=stackdriver
```